index.js

```diff
@@ -1,11 +1,50 @@
 'use strict';
 
 /**
- * Adds commas to a number
- * @param {number} number
- * @param {string} locale
+ * Push and pop from a stack
+ * @param {string} value
+ * @param {number} value
  * @return {string}
+ * @return {array}
  */
-module.exports = function(number, locale) {
-    return number.toLocaleString(locale);
-};
+
+//  creating a stack of length 5
+let lengthOfStack = 5
+let stack = new Array(lengthOfStack);
+let lastUpdatedIndex = 0;
+
+let addValueToStack = (value) => {
+// pushing value onto the stack
+    if (lastUpdatedIndex === lengthOfStack){
+        return "Stack is full, cannot push."
```

Comment:
return type is different, in cases like these there are only two options:
- either return the original stack
- raise an exception

- returning the original stack is misleading to someone using your library so raising an exception is better

Reply:
used throw instead of return for exception statement. at top of the file, @throw {string} has also been added for reference.